modules/microshift-uninstalling-lvms-csi-snapshot.adoc

```diff
@@ -0,0 +1,30 @@
+// Module included in the following assemblies:
+//
+// * microshift_storage/microshift-storage-plugin-overview.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="microshift-uninstalling-lvms-csi-snapshot_{context}"]
+= Uninstalling the CSI snapshot implementation
+
+To uninstall the installed CSI snapshot implementation, use the following procedure.
+
+.Prerequisites
+
+* {microshift-short} is installed and running.
+* The CSI snapshot implementation is deployed on the MicroShift cluster.
+
+.Procedure
+
+. Uninstall the CSI snapshot implementation by running the following command:
++
+[source,terminal]
+----
+$ oc delete -n kube-system deployment.apps/csi-snapshot-controller deployment.apps/csi-snapshot-webhook
```

Comment:
@dhensel-rh should the command should be "oc delete -n kube-system deployment.apps/snapshot-controller deployment.apps/snapshot-webhook"

Reply:
done.